pysollib/games/harp.py

```diff
@@ -333,6 +337,69 @@ def startGame(self):
     getQuickPlayScore = Game._getSpiderQuickPlayScore
 
 
+# ************************************************************************
+# * Churchill Solitaire
+# ************************************************************************
+# https://boardgames.stackexchange.com/questions/29254/rules-for-churchill-solitaire
+
+class Churchill_DevilStack(OpenStack):
+    pass
+
+class Churchill_RowStack(KingAC_RowStack):
+    def acceptsCards(self, from_stack, cards):
+        if isinstance(from_stack, Churchill_DevilStack):
+            return False
+        if isinstance(from_stack, AbstractFoundationStack):
+            return False
+        return KingAC_RowStack.acceptsCards(self, from_stack, cards)
+
+class Churchill_TalonStack(DealRowTalonStack):
+    def dealCards(self, sound=False):
+        def isKingPile(cards):
+            return len(cards) > 0 and \
+                cards[0].rank == KING and \
+                isAlternateColorSequence(cards)
+        rows = [r for r in self.game.s.rows if not isKingPile(r.cards)]
+        return self.dealRowAvail(rows=rows, sound=sound)
+
+class Churchill(Game):
+    Layout_Method = staticmethod(Layout.harpLayout)
+    Talon_Class = Churchill_TalonStack
+    Foundation_Class = SS_FoundationStack
+    RowStack_Class = Churchill_RowStack
+
+    shallHighlightMatch = Game._shallHighlightMatch_AC
+
+    def createGame(self, **layout):
+        # create layout
+        l, s = Layout(self), self.s
+        kwdefault(layout, rows=10, waste=0, texts=1, playcards=30, reserves=1)
+        self.Layout_Method(l, **layout)
+        self.setSize(l.size[0], l.size[1])
+        # create stacks
+        s.talon = self.Talon_Class(l.s.talon.x, l.s.talon.y, self)
+        for r in l.s.foundations:
+            s.foundations.append(
+                self.Foundation_Class(r.x, r.y, self, suit=r.suit))
+        for r in l.s.rows:
+            s.rows.append(self.RowStack_Class(r.x, r.y, self))
+        for r in l.s.reserves:
+            stack = Churchill_DevilStack(r.x,r.y,self)
```

Comment:
Please add a space after each comma in this line (see https://www.flake8rules.com/rules/E231.html)